app/code/Magento/Integration/Plugin/Model/AdminUser.php

```diff
@@ -33,15 +34,18 @@ public function __construct(
      * @param \Magento\User\Model\User $subject
      * @param \Magento\Framework\DataObject $object
      * @return \Magento\User\Model\User
-     * @throws \Magento\Framework\Exception\LocalizedException
      */
     public function afterSave(
         \Magento\User\Model\User $subject,
         \Magento\Framework\DataObject $object
     ): \Magento\User\Model\User {
         $isActive = $object->getIsActive();
         if ($isActive !== null && $isActive == 0) {
-            $this->adminTokenService->revokeAdminAccessToken($object->getId());
+            try {
+                $this->adminTokenService->revokeAdminAccessToken($object->getId());
+            } catch (LocalizedException $e) {
+                //token for admin use doesn't exist in oauth_token table, no action
```

Comment:
This exception looks like intentionally not caught before. What are other possibilities of fix?